internal/namespaces/feedback/custom.go

```
@@ -0,0 +1,91 @@
+package feedback
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"reflect"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/human"
+)
+
+func GetCommands() *core.Commands {
+	return core.NewCommands(
+		feedbackRoot(),
+		feedbackBugCommand(),
+		feedbackFeatureRequestCommand(),
+	)
+}
+
+func feedbackRoot() *core.Command {
+	return &core.Command{
+		Short:     `Send feedback to the Scaleway CLI Team!`,
+		Namespace: "feedback",
+		ArgsType:  reflect.TypeOf(struct{}{}),
+		ArgSpecs:  core.ArgSpecs{},
+	}
+}
+
+func feedbackBugCommand() *core.Command {
+	return &core.Command{
+		Short:     `Send a bug-report`,
+		Long:      `Send a bug-report to the Scaleway CLI team.`,
+		Namespace: "feedback",
+		Resource:  `bug`,
+		ArgsType:  reflect.TypeOf(struct{}{}),
+		ArgSpecs:  core.ArgSpecs{},
+		NoClient: true,
+
+		Run: func(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+			buildInfoStr, err := human.Marshal(core.ExtractBuildInfo(ctx), nil)
+			if err != nil {
+				return nil, err
+			}
+			issue := issue{
+				IssueTemplate: bug,
+				BuildInfoStr:  buildInfoStr,
+			}
+			err = issue.openInBrowser(ctx)
+			if err != nil {
+				log.Fatal(err)
```

You should return the error and let the CLI print the error